src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -22,7 +22,9 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
-
+{% if isColumnLocationDisplayed is not defined %}
+  {% set isColumnLocationDisplayed=(product.availableQuantity is empty) %}
```

Comment:
So the column is displayed when there is no stock? Shouldn't it be the opposite?

Reply:
In products.html.twig you check `product.location is not empty` which seems more logical

Reply:
I guess you add the `isColumnLocationDisplayed ` here because when adding a product the variable is not defined like it is in the products loop
The it would be cleaner to define this variable in the addProduct controller

Reply:
The variable `product` is added in  `src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php` at l.542

Reply:
Fixed for the bad variable used